django/db/models/sql/compiler.py

```diff
@@ -1447,6 +1450,9 @@ def explain_query(self):
             else:
                 yield row
 
+    def comments_sql(self):
+        return " ".join([f"/* {comment} */" for comment in self.query.comments])
```

Comment:
can we avoid multiple calls to `" ".join()` and list comps by putting the comments into each method's `result` list? should speed things up a bit

we could also pre-compute adding the `/*` and `*/` ends when we put strings into `self.query.comments`. This way if a queryset is reused, we don't need to re-do that work each time.

Reply:
I already suggested such changes, but Mariusz requested adding a hook which reverted them.